build-configs/configs/integreat-test-cms/index.js

```diff
@@ -13,6 +14,14 @@ const IntegreatTestCmsBuildConfig: BuildConfigType = {
     pois: true,
     newsStream: true,
     introSlides: true
+  },
+  android: {
+    applicationId: 'tuerantuer.app.integreat',
```

Comment:
It is not possible to change the application id, right? It would be nice to have the same value.

Reply:
No, we have to keep this separate unfortunately :/